src/components/Tag/Tag.tsx

```diff
@@ -29,12 +30,23 @@ export interface ITagProps
 		background (e.g. in a page header). */
 	hasLightBackground: boolean;
 
+	/** Allows Tag to be interacted with via the onClick function */
+	isClickable?: boolean;
```

Comment:
I believe we normally use the existence of the onClick prop to mean the component is clickable. @yasskin can you confirm this?

Reply:
We use an isClickable prop for icons

Reply:
I think this works. It makes the entire tag clickable. I just wonder if the intent of the design of the component is to only have the CloseIcon clickable, not the entire tag.
Take a look at the Interactive story: https://docspot.adnxs.net/projects/anx-react/latest/?path=/docs/ludic-design-system_communication-tag--interactive

Reply:
> I believe we normally use the existence of the onClick prop to mean the component is clickable. @yasskin can you confirm this?

The `isClickable` prop and the pattern used in this PR is used all over the place in the repo, in Icons, for example.

Reply:
@yasskin I think that was the original design of the component, but I'm guessing @lennoxdavidjr needs the whole tag to be clickable for something related to the global search update? There should be a story showing how the props are used though